test/simulator/misc/megaevolution.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Mega Evolution', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it('should overwrite normally immutable abilities', function () {
+		battle = common.createBattle();
+		const p1 = battle.join('p1', 'Guest 1', 1, [{species: "Metagross", ability: 'zenmode', item: 'metagrossite', moves: ['metalclaw']}]);
```

Comment:
Zen Mode??? You had 9 options and you went off script!

Reply:
This was originally a loop that tested all nine but then Zarel mentioned offhand that he didn’t like those so I panicked and changed it to the first forme-change ability I could think of.